source_files/edge/rad_pars.cc

```diff
@@ -917,8 +918,11 @@ static void RAD_ParseTag(param_set_t& pars)
 {
 	// Tag <number>
 
-	if (this_rad->tag != 0)
-		RAD_Error("Script already has a tag: '%d'\n", this_rad->tag);
+	if (this_rad->tag[0] != 0)
+		RAD_Error("Script already has a tag: '%d'\n", this_rad->tag[0]);
+
+	if (this_rad->tag[1] != 0)
+		RAD_Error("Script already has a tag: '%d'\n", this_rad->tag[1]); // Would be more meaningful to show string :/
```

Comment:
We could add a std::string tag_name; to rad_script_s which is set at parse, and output this here instead of the hash

Reply:
This should be doable; will have to touch the save/load functions to make sure it is preserved there as well

Reply:
Other than testing, this looks solid.  Having this message be the tag name would be of assistance to mappers, so if not too bad would be a nice add.  Otherwise, maybe capture in the task tracker?

Reply:
I think it should be doable without much of a headache; I've had some experience converting std::strings to/from a save-friendly format when we fixed monsters not being able to have more than one "wait until death" script apply to them.

Reply:
Doesn't this get parsed whether from a save game or not?  Could just have a static unordered_map of uint32_t => std::string in file if so, and lookup.

Reply:
That's a good question, honestly. If it's always parsed fully from RSCRIPT then it wouldn't need saving

Reply:
Cool, saw the commit to display the tag name, nice!